### ISSUE TITLE
Update OptimizationSolver to use new hyperparameters settings

### ISSUE THREAD

**Issue by AlessandroPierro** [lava-nc/lava-optimization#148]
Objective of issue: hyperparameters settings should be updated in OptimizationSolver tests to match new neuron model

**Lava version:**
- [x] **0.2.3** (current version)

**I'm submitting a ...** 

- [x] tests request

**Comment by GaboFGuerra** [comment]
These should not only be updated on the tests, the solver needs to also assert only valid fields for hyperparameter are allowed

**Comment by AlessandroPierro** [comment]
> These should not only be updated on the tests, the solver needs to also assert only valid fields for hyperparameter are allowed

Agree, we need to design a system to do that since different neuron models accept different hyperparameters.
